sql/updates/world/2024_01_09_fix_npc.sql

```diff
@@ -0,0 +1,136 @@
+DELETE FROM creature WHERE guid IN (344520, 344521, 344522, 344523, 344524, 344525, 344526, 344527, 344528, 344529, 344530, 344531, 344532, 344533, 344534, 344535, 344536, 344537, 344538, 344539, 344540, 344541, 344542, 344543, 344544, 344545, 344546, 344547, 344548, 344549, 344550, 344551, 344552, 344553, 344554, 344555, 344556, 344557, 344558, 344559, 344560, 344561, 344562, 344563, 344564, 344565, 344566, 344567, 344568, 344569, 344570, 344571, 344572, 344573, 344574, 344575, 344576, 344577);
```

Comment:
These creatures are in which zone and area?

Reply:
yes